pkg/distributor/distributor_test.go

```diff
@@ -4835,6 +4835,8 @@ type prepConfig struct {
 	ingestStorageEnabled    bool
 	ingestStoragePartitions int32 // Number of partitions. Auto-detected from configured ingesters if not explicitly set.
 	ingestStorageKafka      *kfake.Cluster
+
+	disableStreamingResponse bool
```

Comment:
Might be worth adding a comment here to explain why we'd want to do this (ie. to simulate a response from an old ingester)

Reply:
Comment added. Thank you.